frontend/src/scenes/notebooks/Notebook/InlineMenu.tsx

```diff
@@ -0,0 +1,80 @@
+import { LemonInput } from '@posthog/lemon-ui'
+import { Editor } from '@tiptap/core'
+import { BubbleMenu } from '@tiptap/react'
+import clsx from 'clsx'
+import { IconBold, IconDelete, IconItalic, IconLink, IconOpenInNew } from 'lib/lemon-ui/icons'
+
+export const InlineMenu = ({ editor }: { editor: Editor }): JSX.Element => {
+    const { href, target } = editor.getAttributes('link')
+
+    const setLink = (href: string): void => {
+        editor.commands.setMark('link', { href: href })
+    }
+
+    const openLink = (): void => {
+        window.open(href, target)
+    }
+
+    return (
+        <BubbleMenu editor={editor} tippyOptions={{}}>
+            <div className="NotebookInlineMenu flex bg-white rounded border items-center text-muted-alt p-0.5">
+                {editor.isActive('link') ? (
+                    <>
+                        <LemonInput
+                            size="small"
+                            placeholder="https://posthog.com"
+                            onChange={setLink}
+                            value={href}
+                            className="border-0"
+                            autoFocus
+                        />
+                        <Option onClick={openLink} status="primary">
+                            <IconOpenInNew />
+                        </Option>
+                        <Option onClick={() => editor.commands.unsetMark('link')} status="destructive">
+                            <IconDelete />
+                        </Option>
+                    </>
+                ) : (
+                    <>
+                        <Option onClick={() => editor.commands.toggleMark('bold')} active={editor.isActive('bold')}>
+                            <IconBold />
+                        </Option>
+                        <Option onClick={() => editor.commands.toggleMark('italic')} active={editor.isActive('italic')}>
+                            <IconItalic />
+                        </Option>
+                        <Option onClick={() => editor.commands.setMark('link')}>
+                            <IconLink />
+                        </Option>
+                    </>
+                )}
+            </div>
+        </BubbleMenu>
+    )
+}
+
+const Option = ({
+    status = 'default',
+    active = false,
+    children,
+    onClick,
+}: {
+    status?: 'default' | 'primary' | 'destructive'
+    active?: boolean
+    children: JSX.Element
+    onClick: () => void
+}): JSX.Element => {
+    return (
+        <button
```

Comment:
These field and look weird - could we not use LemonButton in some format? 

<img width="188" alt="Screenshot 2023-09-21 at 09 53 24" src="https://github.com/PostHog/posthog/assets/2536520/480934ff-6272-449e-8de8-6ac27a71009e">

Also when you click it deselects the text. I wonder if there is a way of keeping it highlighted? 

![2023-09-21 09 53 29](https://github.com/PostHog/posthog/assets/2536520/b33a499b-7ffd-4e5a-8b19-8e19a47b4036)

Reply:
I originally tried LemonButton but backed out of it because I was making too many customizations in CSS. I've come full circle to something much nicer.

I thought the deselecting text was a bug in TipTap but someone pointed out on my issue that I just needed to refocus: https://github.com/ueberdosis/tiptap/issues/4462